CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.33.0]
+
+### Added
+- Support for providing custom `aria-label` when ListBox is used.
+
+### Fixed
+- Updating the markers on live streams causing unhandled exception after player is destroyed
```

Comment:
For consistency with previous changelog entries.

```suggestion
- Updating the markers on live streams causing unhandled exception after player is destroyed.
```

Reply:
Some have it, some don't. But sure :)